libde265/x86/CMakeLists.txt

```diff
@@ -13,11 +13,13 @@ add_library(x86_sse OBJECT ${x86_sse_sources})
 set(sse_flags "")
 
 if(NOT MSVC)
-  set(sse_flags "${sse_flags} -msse4.1")
+  if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(sse_flags "${sse_flags} -msse4.1")
+  else(CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
Can the 'else' branch ever be executed?

Reply:
Yes, if compiling in 32-bit platforms (repeating the clause is a matter of CMake notation).